scripts/utilities/update_revision.sh

```diff
@@ -121,14 +121,18 @@ print_msg ()
 #
 # Parameters:
 #    $1 - sources config path
+#    $2 - package
+#    $3 - configset
```

Comment:
Maybe also update the description of `$1`, as it expects the repo's URL, instead of sources config path.